homeassistant/components/openai_conversation/const.py

```diff
@@ -0,0 +1,29 @@
+"""Constants for the OpenAI Conversation integration."""
+
+DOMAIN = "openai_conversation"
+CONF_PROMPT = "prompt"
+DEFAULT_MODEL = "text-davinci-003"
+DEFAULT_PROMPT = """
+You are a conversational AI for a smart home named {{ ha_name }}.
+If a user wants to control a device, reject the request and suggest using the Home Assistant UI.
+
+The people living in this smart home are:
```

Comment:
Won't this mean we send the names of our users to a cloud service?

Reply:
correct. You need to feed the AI info or else you can't ask about it.

Reply:
Right, but this is feeding it personal details of our users, by default, without asking the user. That doesn't go along with our principal values, even if this integration is optional, I think.

Reply:
What if you swap the name for a template (such as `#NAME#`) and have it be swapped for the actual name when processing the response?

Reply:
The goal of the conversation agent is to be able to answer the users questions, including something like "where is Paulus".

There are different types of clouds:

![CleanShot 2023-01-25 at 09 59 26](https://user-images.githubusercontent.com/1444314/214597365-705d0079-5ceb-4504-9cf9-8ea4c7777be6.png)

In this case it's a generic cloud that users choose to configure.